ramls/loan-policy.json

```diff
@@ -44,6 +44,12 @@
         "fixedDueDateScheduleId": {
           "type": "string",
           "description": "Fixed due date schedule (due date limit)"
+        },
+        "itemLimit": {
```

Comment:
What is the item limit? Is it the number of items a patron can have checked out at the same time?

Reply:
Hi @marcjohnson-kint, please find a detailed explanation of what an `Item limit` is in the description of [CIRC-558](https://issues.folio.org/browse/CIRC-558).